evap/staff/importers.py

```diff
@@ -318,13 +315,13 @@ def process(cls, excel_file, semester, vote_start_date, vote_end_date, test_run)
             importer = cls()
             importer.read_book(excel_file)
             if importer.errors:
-                return importer.success_messages, importer.warnings, importer.errors
+                return importer.success_messages, dict(importer.warnings), importer.errors
```

Comment:
okay, in that case i'd do the conversion where this thing is passed to the template.